### ISSUE TITLE
Add Hearings Page: content behind modal is read by a screen reader, but shouldn't be.

### ISSUE THREAD

**Issue by alisan16** [department-of-veterans-affairs/caseflow#12121]
If a user selects the Add hearing Date element, a modal window is displayed visually. Page content can be accessed by assistive technology users when this modal window is displayed. Visually users know a modal is displayed and cannot access other features in the background while this window has focus. (New: 08/27/2019) you can see in the screen shot below that focus is on the “digital service at VA” verses in the modal box

Suggested fix: https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/dialog.html
